Update README.md (#341)
openjs-foundation/summit: README.md

Join [#collabsummit](https://openjs-foundation.slack.com/archives/CPE10404W) slack channel in [OpenJS Foundation slack](https://slack-invite.openjsf.org/).

## Upcoming Events

> We are sorry there are no upcoming events

Reviewer: ```suggestion
There are no upcoming events at this time.
```
